_includes/configuration/contextmenu_avoid_overlap.md

```diff
@@ -0,0 +1,18 @@
+## `contextmenu_avoid_overlap`
+
+The `contextmenu_avoid_overlap` option allows preventing the context menu overlapping with specific nodes within the editor. This option takes a CSS selector and when the context menu is opened on a node that matches the specified selector, the context menu will render below instead of where the click occurred.
```

Comment:
```suggestion
{{site.requires_5_5v}}

The `contextmenu_avoid_overlap` option prevents the context menu from covering (or overlapping) specific nodes within the editor. This option accepts a [CSS selector](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors). When the context menu is opened on a node that matches the specified selector, the context menu will render below the node, instead of where the click occurred.
```

Reply:
Fixed/accepted 👍